src/utils/symbol_utils.ml

```diff
@@ -147,3 +147,17 @@ let get_program_entry_points program =
   let entry_points = Seq.filter subfunctions ~f:(fun subfn -> Term.has_attr subfn Sub.entry_point) in
   let main_fn = Seq.filter subfunctions ~f:(fun subfn -> "@main" = Tid.name (Term.tid subfn)) in
   Seq.append main_fn entry_points
+
```

Comment:
These three functions should not be here. The module Symbol_utils deals with imported/exported ELF symbols.
Since these functions are all machine related, there could be a module Machine_utils or something similar.